--- C:/Program Files (x86)/robotology/opencv-3.4.2-v14-x86/OpenCVConfig.cmake
+++ C:/Program Files (x86)/robotology/opencv-3.4.2-v14-x86/OpenCVConfig.cmake
@@ -105,17 +105,21 @@
 
 get_filename_component(OpenCV_CONFIG_PATH "${CMAKE_CURRENT_LIST_FILE}" PATH)
 if(OpenCV_RUNTIME AND OpenCV_ARCH)
-  if(OpenCV_STATIC AND EXISTS "${OpenCV_CONFIG_PATH}/${OpenCV_ARCH}/${OpenCV_RUNTIME}/staticlib/OpenCVConfig.cmake")
+  # broke in windows see https://github.com/conda-forge/opencv-feedstock/issues/41
+  if(OpenCV_STATIC AND EXISTS "${OpenCV_CONFIG_PATH}/lib/OpenCVConfig.cmake")
+    # cuda not available in windows see https://github.com/conda-forge/opencv-feedstock/issues/109
     if(OpenCV_CUDA AND EXISTS "${OpenCV_CONFIG_PATH}/gpu/${OpenCV_ARCH}/${OpenCV_RUNTIME}/staticlib/OpenCVConfig.cmake")
       set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/gpu/${OpenCV_ARCH}/${OpenCV_RUNTIME}/staticlib")
     else()
-      set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/${OpenCV_ARCH}/${OpenCV_RUNTIME}/staticlib")
+      set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/lib")
     endif()
-  elseif(EXISTS "${OpenCV_CONFIG_PATH}/${OpenCV_ARCH}/${OpenCV_RUNTIME}/lib/OpenCVConfig.cmake")
+  # shared libs (dlls)
+  elseif(EXISTS "${OpenCV_CONFIG_PATH}/lib/OpenCVConfig.cmake")
+    # cuda not available in windows see https://github.com/conda-forge/opencv-feedstock/issues/109
     if(OpenCV_CUDA AND EXISTS "${OpenCV_CONFIG_PATH}/gpu/${OpenCV_ARCH}/${OpenCV_RUNTIME}/lib/OpenCVConfig.cmake")
       set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/gpu/${OpenCV_ARCH}/${OpenCV_RUNTIME}/lib")
     else()
-      set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/${OpenCV_ARCH}/${OpenCV_RUNTIME}/lib")
+      set(OpenCV_LIB_PATH "${OpenCV_CONFIG_PATH}/bin")
     endif()
   endif()
endif()
